lambdas/checklist/serverless.yml

```diff
@@ -71,5 +70,16 @@ functions:
               paths:
                 id: true
 
+  publish-evaluation:
+    name: ${self:provider.stage}-checklist-publish-evaluation
```

Comment:
Let's update this to:
name: ${self:provider.stage}-checklist-evaluation-publisher

too